pkg/template/cluster/cluster.go

```diff
@@ -68,6 +69,23 @@ func newClusterCR(obj interface{}, clusterID string, c Config) (*apiv1alpha2.Clu
 		return nil, microerror.Mask(err)
 	}
 
+	clusterLabels := map[string]string{}
+
+	for key, value := range c.Labels {
+		clusterLabels[key] = value
+	}
+
+	gsLabels := map[string]string{
+		label.ClusterOperatorVersion: c.ReleaseComponents["cluster-operator"],
+		label.Cluster:                clusterID,
+		label.Organization:           c.Owner,
+		label.ReleaseVersion:         c.ReleaseVersion,
```

Comment:
Is there any reason for not defining these as the initial values inside the `clusterLabels` map?

Reply:
The idea was that (although possibly prevented through the check for forbidden label keys), to always have the required labels overwrite userlabels.